src/v2/System/SystemContext.tsx

```diff
@@ -111,10 +113,21 @@ export const SystemContextProvider: FC<SystemContextProps> = ({
     isLoggedIn: !!user,
   }
 
+  const localizations = {
+    en_US: locale_en_us,
+  }
+
   return (
-    <SystemContext.Provider value={providerValues}>
-      {children}
-    </SystemContext.Provider>
+
+    <IntlProvider
+      messages={localizations["en_US"]}
+      locale="en-us"
+      defaultLocale="en-us"
+    >
+      <SystemContext.Provider value={providerValues}>
+        {children}
+      </SystemContext.Provider>
+    </IntlProvider>
```

Comment:
This can be removed in favor of the Boot suggestion up above

Reply:
Although we _will_ be using system context for triggering updates to the language. 

```tsx
const { setLanguage } = useSystemContext()
... 

<Select values={languages} onChange={lang => setLanguage(lang) />
```

That kind of thing.

Reply:
I made the changes to create an `InternationalizationProvider` and I'm importing it in `Boot`, but it's still not really clear to me how can I remove this from `SystemContext` without affecting the tests... 🤔

Reply:
It might impact the tests, but in general all of our react context providers live in boot since that's the root of our application. For tests, if we need access to certain providers, we can wrap the test in `<MockBoot>`

Reply:
Cool, thank you!!
I wrapped NavBar tests in `<MockBoot>` and those are passing now! 🙌 

Lack of `IntlProvider` in `SystemContext` is leading to issues on `/terms`, though... is this  one an exception that doesn't benefit from `Boot`?

<img width="1764" alt="Screenshot 2022-03-11 at 18 36 57" src="https://user-images.githubusercontent.com/8002618/157919803-f24aeaf3-70a2-401a-9198-ba7500e52b71.png">

Reply:
This started to happen after removing `IntlProvider` from `SystemContext` because `/terms` does not rely on `Boot`. As we can see [in the `.jade` files](https://github.com/artsy/force/blob/9e9af2114ff29b0565e10b88a80e9e081afec954/src/desktop/components/main_layout/header/templates/index.jade), we use some stich components, which replicate parts of the components using on static pages like these.

In order to make this work, we should also add the `IntlProvider` to the components in the [stitch_components ](https://github.com/artsy/force/tree/main/src/desktop/components/react/stitch_components) directory (components that can't benefit from `Boot`)